main.go

```diff
@@ -168,6 +168,13 @@ func main() {
 	// Subscribe to LND invoice updates in the background
 	go svc.InvoiceUpdateSubscription(context.Background())
 
+	//Start webhook subscription
+	if svc.Config.WebhookUrl != "" {
+		webhookCtx, cancelWebhook := context.WithCancel(context.Background())
+		go svc.StartWebhookSubscribtion(webhookCtx)
+		defer cancelWebhook()
```

Comment:
Do I understand this Go correct: 
Sending the webhook basically happens asynchronously in its own webhook context? 
This means that the HTTP response finishes/returns and the webhook still executes?

Reply:
Yes, this is totally seperate from the main server. LNDhub has it's own Pubsub system where `models.Invoice` objects are being passed around that represent ingoing and outgoing invoices.

Reply:
When you make a `Pay` call, the publishing part of the PubSub system is executed synchronously, but all possible subscribers are executed asynchronously.